### ISSUE TITLE
[iOS][Accessibility] Cover Block: Fix selecting background image with VoiceOver on

### ISSUE THREAD

**Issue by Tug** [wordpress-mobile/gutenberg-mobile#2699]
When first adding a Cover block and selecting it, it takes you into adding a paragraph block rather than giving you the option to add an image immediately as the block does when Voice Over is off. Here’s a video of the issue:

https://d.pr/v/jllfUq

If you try to work around the issue with the Cover block directing you to add a paragraph block inside the cover block (instead of an image) using the block navigation toolbar to “Navigate Up” in the block hierarchy to select the cover block itself, there is an image icon that appears in the upper right of the cover block that can be selected. Voice Over mentions the button when describing the contents of the block (button, image), but when the button is actually selected, it does not say anything. This button is the button that gives you the option to select an image or video for use with the cover block, so the fact it isn’t spoken when selected is a bit of a problem. Here’s a video of this issue:

https://d.pr/v/sQj2Me

**To Reproduce**
1. With VoiceOver enabled
2. Open the WordPress app (or the demo app)
3. Create a new post
4. Open the block inserter
5. Select and add a cover block
6. Select the cover block
7. Notice you're invited to edit text inside a paragraph instead of picking between adding a background image or editing text

**Expected behavior**
Selecting a cover block which does not have a background set should invite the user to pick a background image first.

**Smartphone (please complete the following information):**
 - Device: iPhone
 - OS: all iOS versions

**Additional context**
Reported by Galen, noticed in a user ticket: 3316181-zen
